app/assets/javascripts/listings/ListingEligibilityService.js.coffee

```diff
@@ -32,17 +32,20 @@ ListingEligibilityService = ($localStorage, ListingIdentityService, ListingUnitS
     else
       parseFloat(Service.eligibility_filters.income_total)
 
-  Service.occupancyMinMax = (listing) ->
+  Service.occupancyMinMax = (listing, amiLevel) ->
     minMax = [1, 1]
     if listing.unitSummary
       min = _.min(_.map(listing.unitSummary, 'minOccupancy')) || 1
-      max = _.max(_.map(listing.unitSummary, 'maxOccupancy')) || 2
+      max = _.max(_.map(listing.unitSummary, 'maxOccupancy')) || Service.defaultMaxOccupancy(amiLevel)
       minMax = [min, max]
     return minMax
 
+  Service.defaultMaxOccupancy = (amiLevel) ->
```

Comment:
not super clear on what this is. It's saying that the max is whatever the max value is in the AMI table?